modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -105,4 +130,18 @@ data:
 <1> Specifies the name of the ApplicationSet GitLab SCM Provider.
 <2> Specifies the namespace where the ApplicationSet GitLab SCM Provider is defined.
 +
-For more information, see link:https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#add-self-signed-tls-certificate-for-gitlab-scm-provider-to-applicationsets-controller[Add Self-signed TLS Certificate for Gitlab SCM Provider to ApplicationSet Controller].
\ No newline at end of file
+For more information, see link:https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#add-self-signed-tls-certificate-for-gitlab-scm-provider-to-applicationsets-controller[Add Self-signed TLS Certificate for Gitlab SCM Provider to ApplicationSet Controller].
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value are no longer classified as global secrets applicable to all `Applications` and `ApplicationSets`. 
++
+In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL. 
```

Comment:
```suggestion
In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` resource could use a cluster secret that matches the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets scoped to an application's project will be considered for matching a repository URL. 
```

Reply:
I've checked with the SME and he is in agreement with the modification suggestion. Fixed.

Reply:
Fixed.